core/src/main/resources/lib/layout/breadcrumbBar.jelly

```diff
@@ -34,42 +34,38 @@ THE SOFTWARE.
     use the &lt;l:breadcrumb> tag.
   </st:documentation>
 
-  <tr id="top-nav">
```

Comment:
Why did you remove this? Did you perform search across the Jenkinsci org to see how this would affect plugins?

Reply:
Also, if there's a `<tr>`, there must be a wrapping table somewhere.

Reply:
There used to be until that table-div migration.

Reply:
It should be unrelated, as table-div migration is scoped to just form elements

Reply:
Well, I only noticed this because the tr and td weren't showing up in the DOM inspector, and I noticed in the view source tab that those elements were highlighted as syntax errors as it was wrapped in a div. The breadcrumb bar is only invoked from the main layout pages as far as I can tell. I was originally hoping to remove the use of YUI in this widget, but then I got distracted by the ARIA stuff.

Reply:
It should be a separate PR IMO

Reply:
There is no wrapping table anymore since 2014 https://github.com/jenkinsci/jenkins/commit/f277502a8304e38f51160897b9cb2e4ed9910e2c so removing the tr/td here seems fine (the browser just ignores those, so any plugins depending on `#top-nav` selector being in DOM would fail anyway). That being said maybe the one CSS reference to `#top-nav` in core can also be removed in this PR.